articles/aks/node-autoprovision.md

```diff
@@ -64,11 +64,22 @@ NAP is based on the Open Source [Karpenter](https://karpenter.sh) project, and t
     ```
 
 ## Limitations
-
-- Windows and Azure Linux node pools aren't supported yet
-- Kubelet configuration through Node pool configuration is not supported
-- NAP can only be enabled on new clusters currently
-- It is not currently possible to stop nodepools or clusters which use the NAP feature
+### Unsupported features:
+- Windows and Azure Linux node pools
+- Kubelet configuration through Node pool configuration
+- IPv6 clusters
+- [Service Principals](./kubernetes-service-principal.md)
+> [!NOTE]
+  > You can use either a system-assigned or user-assigned managed identity
+- Disk encryption sets
+- CustomCATrustCertificates
+- [Start Stop mode](./start-stop-cluster.md)
+- [HTTP proxy](./http-proxy.md)
+- [OutboundType](./egress-outboundtype.md) mutation (all OutboundTypes are supported, you just cannot change them once created)
+
+### Other limitations
```

Comment:
Upon further review, I think this section isn't necessary. I suggest moving the two bullet points under the heading "## Limitations" and then following it you have the sub-section "###Unsupported features".

Reply:
Then you can remove "### Other limitations"

Reply:
sg